### PR TITLE
Update storage.routing.ts

### PR DESCRIPTION
**Pull request by MDVAurelien**
Add T markers to obtain matches in translation files,

### CODE REVIEW

File: src/app/pages/storage/storage.routing.ts [hunk 1]
```diff
@@ -1,27 +1,30 @@
...
+import { marker as T } from '@biesbjerg/ngx-translate-extract-marker';
+import { TranslationsLoadedGuard } from 'app/core/guards/translations-loaded.guard';
 import { PoolsDashboardComponent } from 'app/pages/storage/components/pools-dashboard/pools-dashboard.component';
 import { ManagerComponent } from './components/manager/manager.component';
 
 export const routes: Routes = [
   {
     path: '',
-    data: { title: 'Storage' },
+    canActivate: [TranslationsLoadedGuard],
```
Comment (undsoft) [inline src/app/pages/storage/storage.routing.ts:11]: This isn't necessary, as this guard is applied on a level higher in `app.routes.ts`.

**Review by undsoft — CHANGES_REQUESTED** [review]
It would also be good to execute `yarn extract` to regenerate translation files.
If you don't have yarn, we can skip it and merge as is and wait for somebody else to have them regenerated in their PRs in the future.

**Comment by MDVAurelien** [comment]
> 



> yarn extract

Thks, how i can do a yarn extract ?

**Comment by undsoft** [comment]
Do you have node on your system? If so, you can install yarn https://classic.yarnpkg.com/lang/en/docs/install/#mac-stable
and then execute `yarn` and `yarn extract` in the terminal in the directory with the project.

**Comment by MDVAurelien** [comment]
> Do you have node on your system? If so, you can install yarn https://classic.yarnpkg.com/lang/en/docs/install/#mac-stable and then execute `yarn` and `yarn extract` in the terminal in the directory with the project.

No, but I'll look into it. For the moment you can validate the commit, someone will launch a yarn extract in a PR later. Thks Evgeny.

**Review by undsoft — APPROVED** [review]
👍🏻

**Pull request merged**